exercises/concept/lasagna/.docs/instructions.md

```diff
@@ -0,0 +1,43 @@
+# Instructions
+
+Lucian's girlfriend is on her way home and he hasn't cooked their anniversary dinner!
+
+In this exercise, you're going to write some code to help Lucian cook an exquisite lasagna from his favorite cookbook.
+
+You have four tasks, all related to the time spent cooking the lasagna.
+
+## 1. Define the expected oven time in minutes
+
+Define the `ovenTime()` method that does not take any parameters and returns how many minutes the lasagna should be in the oven. According to the cooking book, the expected oven time in minutes is 40:
```

Comment:
here and in the rest of the file: "method" -> "function"
One line per sentence.
"cooking book" -> "cookbook"
maybe "expected oven time in minutes is 40" -> "expected oven time 40 minutes"